README.md

```diff
@@ -75,10 +55,26 @@ If you are editing the API definitions, generate the manifests such as CRs or CR
 make manifests
 ```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+**NOTE:** Run `make help` for more information on all potential `make` targets
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+### Test strategy
+
+Run unit tests
```

Comment:
If the tests should run one after, then change the steps to the ordered list. If not, then leave it as it is.
```suggestion
1. Run unit tests
```